src/python/pants/backend/jvm/tasks/jvm_compile/rsc/rsc_compile.py

```diff
@@ -581,6 +583,7 @@ def create_compile_context(self, target, target_workdir):
         args_file=os.path.join(zinc_dir, 'zinc_args'),
         post_compile_merge_dir=os.path.join(zinc_dir, 'post_compile_merge_dir'),
         sources=sources,
+        macros_in_dependency_classpath_file=os.path.join(target_workdir, 'macros-in-classpath'),
```

Comment:
Nit: It might be worth updating the layout comment above, it may not be. It's out of date anyway, afaict.